TPP.Core.Tests/Commands/Definitions/BettingCommandsTest.cs

```diff
@@ -0,0 +1,118 @@
+using System;
+using System.Collections.Immutable;
+using System.Threading.Tasks;
+using Moq;
+using NodaTime;
+using NUnit.Framework;
+using TPP.ArgsParsing;
+using TPP.ArgsParsing.TypeParsers;
+using TPP.Core.Commands;
+using TPP.Core.Commands.Definitions;
+using TPP.Match;
+using TPP.Persistence.Models;
+
+namespace TPP.Core.Tests.Commands.Definitions
+{
+    public class BettingCommandsTest
+    {
+        private static User MockUser(string name) => new User(
+            id: Guid.NewGuid().ToString(),
+            name: name, twitchDisplayName: "☺" + name, simpleName: name.ToLower(), color: null,
```

Comment:
nice
(as a note I don't think emoji in display names will ever come up, but I guess it costs nothing to support them, unsure if it'll work with the db though)